dev/README_RELEASE_PROVIDERS.md

```diff
@@ -696,15 +696,15 @@ subject:
 
 ```shell script
 cat <<EOF
-$([ $VOTE_DURATION_IN_HOURS -ge 72 ] && echo "[VOTE]" || echo "[ACCELERATED VOTE]") Airflow Providers prepared on $(LANG=en_US.UTF-8 TZ=UTC date "+%B %d, %Y")
+$([ $VOTE_DURATION_IN_HOURS -ge 72 ] && echo "[VOTE]" || echo "[ACCELERATED VOTE]") Airflow Providers, package date ${PACKAGE_DATE}")
```

Comment:
why did you remove the lang?
this is needed when release manager OS is not in english by default

Reply:
ah OK since we are having format like: `YYYY-MM-DD` that is fine.
Initially I used lang to avoid confusing between `MM-DD-YYYY` and `DD-MM-YYYY`

Reply:
I guess though we also need to update the `## Summarize the voting for the Apache Airflow release` part with title that matches the package date. And possibly also announcement email?